python/tests/conftest.py

```diff
@@ -103,6 +103,23 @@ def azure_client_and_create_container(azurite_container, azurite_azure_uri):
     return client
 
 
+@pytest.fixture(scope="function")
+def azure_account_sas_token(azure_client_and_create_container, azurite_azure_test_connection_setting):
+    start_time = datetime.now(timezone.utc)
+    expiry_time = start_time + timedelta(days=1)
+    _, _, credential_name, credential_key, _ = azurite_azure_test_connection_setting
+
+    sas_token = generate_account_sas(
+        account_key=credential_key,
+        account_name=credential_name,
+        resource_types=ResourceTypes.from_string("sco"),
+        permission=AccountSasPermissions.from_string("rwdlacup"),
```

Comment:
What do these special strings mean? `sco`, `rwdlacup`? Worth a comment

Reply:
That is a miss. Will do.